tests/org.eclipse.n4js.hlc.tests/test-catalog.json

```diff
@@ -0,0 +1 @@
+{"sessionId":"ee5bb6a5-b7d0-4076-8125-39444c5f970a","testDescriptors":[{"origin":"DemoTest","fqn":"src-gen/BarTest/OsInspectorTest2","testMethods":["testFail"]},{"origin":"DemoTest","fqn":"src-gen/BazTest/OsInspectorTest3","testMethods":["testIgnored"]},{"origin":"DemoTest","fqn":"src-gen/FooTest/OsInspectorTest","testMethods":["testPass"]},{"origin":"DemoTest","fqn":"src-gen/subfolder/SubFolderModule/SubFolderTest","testMethods":["testPass"]},{"origin":"SysProjectA","fqn":"src-gen/T/T","testMethods":["t"]},{"origin":"TestProjectA","fqn":"src-gen/A/A","testMethods":["a"]},{"origin":"TestProjectA","fqn":"src-gen/B/B","testMethods":["b1","b2"]},{"origin":"TestProjectB","fqn":"src-gen/CSub1/CSub1","testMethods":["c1","c2"]},{"origin":"TestProjectB","fqn":"src-gen/CSub2/CSub2","testMethods":["c1","c2","c3"]}]}
```

Comment:
I think you added this file accidentally. Maybe add to `.gitignore` to avoid this from happening in the future.